.github/workflows/rust.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           sed -e s/UID/$UID/ -e s/PATH/path/ CI/dbus-session.conf > /tmp/dbus-session.conf
           sed -e s/UID/$UID/ -e s/PATH/abstract/ CI/dbus-session.conf > /tmp/dbus-session-abstract.conf
           sudo apt-get install -y dbus
+          sudo apt-get install -y ibus
```

Comment:
Nitpick: The containing commit is not touch `zbus` subcrate. So the prefix should be `CI: `.

Reply:
maybe need to start the ibus `ibus start &`

Reply:
I was talking about the git commit message, not the change itself. I forgot that one can add comments to a file as whole (rather than to specific changes in the file). Wish there was a way to do that on commit as well.